led/index.js

```diff
@@ -45,6 +49,11 @@ class RGBLed {
     this._rgb = Color(color).rgb().array();
     return this;
   }
+  
+  getColor() {
+    this._leds.forEach((led, index) => led.getValue());
```

Comment:
Why do you need this line here? :) What do you want the behaviour to be? Is the point of this method to return the RGB values in `this._rgb`? Then you can remove this line. If you want to get the "actual" value from each led, then you can write:

```javascript
let leds = [];
this._leds.forEach((led) => leds.append(led.getValue());
return leds;
```

Reply:
The point is to return the RGB value array. 

I use it as follows to return a hex value to the client via express. 

Trying to reduce the amount of lines that the client needs to do this. 

light = new RGBLed({ red: 12, green: 16, blue: 20 });
res.send('#' + convert.rgb.hex(light.getColor()));

Reply:
Then I think it should be enough to just write `return this._rgb;`. Does that work? :) 
`this._leds.forEach((led, index) => led.getValue());` will not modify anything.